index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Next Generation Scrum Poker</title>
+  </head>
+  <style>
+    :root {
+      --tng-blue: rgb(5, 99, 165);
+    }
+    body {
+      font: 16px sans-serif;
+      display: grid;
+      grid-gap: 0.75rem;
+      grid-template-columns: min-content min-content;
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      margin: 0.5rem;
+      align-content: center;
+      justify-content: center;
+    }
+    div {
+      grid-column: 1/3;
+      grid-row: 1/2;
+    }
+    h1 {
+      color: var(--tng-blue);
+      font-size: 1.5rem;
+      text-align: center;
+      line-height: 1.2;
+      font-weight: normal;
+      white-space: nowrap;
+    }
+    .link {
+      white-space: nowrap;
+      text-decoration: none;
+    }
+    .link:hover {
+      text-decoration: underline;
+    }
+    p {
+      line-height: 1.5rem;
+      text-align: center;
+    }
+  </style>
+  <body>
+    <div>
+      <h1>
+        NEXT GENERATION
+        <br />
+        SCRUM POKER
+      </h1>
+      <p>
+        You are using the legacy version and will be redirected to the new version:
+        <a class="link" href="https://www.scrum-poker.aws.tngtech.com/">
+          https://www.scrum-poker.aws.tngtech.com/
+        </a>
+        within 10s.
```

Comment:
Not sure about this, maybe more like an action item?

```suggestion
        We have moved to a new location! Please update your bookmarks:
        <a class="link" href="https://www.scrum-poker.aws.tngtech.com/">
          https://www.scrum-poker.aws.tngtech.com/
        </a>
        You will be redirected within 10s.
```